src/dev-utils/enableenginedebug.js

```diff
@@ -463,6 +476,22 @@ function enableDocumentTools() {
 		logDocument( this, version );
 	};
 
+	ModelDocument.prototype.initializeDebugging = function() {
```

Comment:
What is the reason for having this as separate function? Couldn't it be a part of `addAppliedDelta` (in `if` statement)?